lib/utils.js

```diff
@@ -50,7 +50,8 @@ function isDomElement(o) {
 }
 
 function isSpecialClick(event) {
-  return event.altKey || event.ctrlKey || event.metaKey || event.shiftKey;
+  let isMiddleClick = (event.button === 1);
```

Comment:
No prob, since we cannot enforce them there's no issue right now. At some point we will need to enforce one or another style